agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -4266,6 +4269,82 @@ bool slave_thread::send_cmdu_to_controller(ieee1905_1::CmduMessageTx &cmdu_tx)
                                   backhaul_params.bridge_mac);
 }
 
+/**
+ * @brief Diffie-Hellman public key exchange keys calculation
+ *        class member params authkey and keywrapauth are computed
+ *        on success.
+ *
+ * @param m1 WSC M1 TLV received from the radio agent
+ * @return true on success
+ * @return false on failure
+ */
+bool slave_thread::autoconfig_wsc_calculate_keys(std::shared_ptr<ieee1905_1::tlvWscM2> m2)
+{
+    auto mac = network_utils::mac_from_string(backhaul_params.bridge_mac);
```

Comment:
Do you mean an explicit field in the DB node structure? If so, I agree - and I guess it can be done as part of the handling of none intel agents, correct?

Reply:
I would make a separate PR of that change. But I can take it, yes.